### DISCUSSION TITLE
TOC title to be diff't than topic title name

### DISCUSSION THREAD

**Discussion by spinningrachel** [squidfunk/mkdocs-material#2151] [Q&A] [answered]
Hi there. 

I'd love to have different topic titles than those that appear in the table of contents, so that if the heading is long for example, I can call it something else in the TOC so it looks better and then give it its full name as the title in the topic itself (the first image). 

![image](https://user-images.githubusercontent.com/13101827/103149919-0d9ceb00-4777-11eb-98fb-219d0dfe7969.png)

I thought if I added meta to the extensions, then the nav title (both in the second image) would render as the TOC title. 

![image](https://user-images.githubusercontent.com/13101827/103149899-cd3d6d00-4776-11eb-8827-2ff518b855db.png)

 and the topic title in the metadata of the topic itself (third image) would render as the topic title, but that's not what's happening.
![image](https://user-images.githubusercontent.com/13101827/103149965-80a66180-4777-11eb-9166-2f636e124e14.png)

I also tried this syntax in the topic markdown:
![image](https://user-images.githubusercontent.com/13101827/103149994-e397f880-4777-11eb-99d9-97d97cf040b1.png)

How *can* I do this, or ... what am I doing wrong? 

Thank you!

**Comment by martinbira** [comment, marked as answer]
So the `title` is for the page meta. But, what you need to do is set the page title as the level 1 title in the markdown:

```
---
title: Set up the Management server
---

# Set up the Management server

Once you've...
```

**Comment by spinningrachel** [comment]
Thanks @martinbira !! Works like a charm :)

**Comment by squidfunk** [comment]
Note that you can also [explicitly define the TOC title/label](https://python-markdown.github.io/extensions/toc/#custom-labels) if you enabled the [Attribute Lists](https://python-markdown.github.io/extensions/attr_list/) extension:

``` markdown
## Document title {: data-toc-label='Table of contents title' }
```

This will leave the [page title](https://github.com/squidfunk/mkdocs-material/blob/afff64df990074b6f58b6d9ba539d273ff909344/src/base.html#L70-L79) as-is, which is probably want you'd want to show up on Google Search results.
